middlewares/authrizeOwnOrSuperUser.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * This middleware authorizes if the requested resource belongs to that user or the user is a superuser
+ * for that route.
+ * Note: This must be added on routes after the `authenticate` middleware.
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ * @param next {Function} - Express middleware function
+ * @return {Object} - return un
```

Comment:
`return un`? not getting what it is.
Please see if there's some typo

Reply:
changing, thanks